content/en/blog/_posts/2021-07-27-csi-windows-support-with-csi-proxy-reaches-ga.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: blog
+title: 'Kubernetes 1.22: CSI Windows Support with CSI Proxy reaches GA'
+date: 2021-07-27
+slug: csi-windows-support-with-csi-proxy-reaches-ga
+---
+
+**Authors:** Mauricio Poppe (Google), Jing Xu (Google), and Deep Debroy (Apple)
+
+*The GA version of CSI Proxy for Windows is being released in Kubernetes 1.22.  CSI Proxy enables CSI Drivers running on Windows nodes to perform privileged storage operations.*
```

Comment:
If this goes out post-release, the text should be:
```suggestion
*The stable version of CSI Proxy for Windows has been released alongside Kubernetes 1.22.  CSI Proxy enables CSI Drivers running on Windows nodes to perform privileged storage operations.*
```

Reply:
ack